libs/langchain/pyproject.toml

```diff
@@ -125,6 +125,8 @@ newspaper3k = {version = "^0.2.8", optional = true}
 amazon-textract-caller = {version = "<2", optional = true}
 xata = {version = "^1.0.0a7", optional = true}
 xmltodict = {version = "^0.13.0", optional = true}
+betabageldb = {version = "0.2.32", optional = true, python = ">=3.7,<3.11"}
```

Comment:
I'm removing betabageldb not sure how it's related to the PR

Reply:
Was in master and then disappeared but noticed some code imported it - thats why i left it in just to be safe.